### ISSUE TITLE
Dependency Issue : org.jetbrains.kotlin:kotlin-stdlib:{strictly 1.3.31}

### ISSUE THREAD

**Issue by sivabe35** [Chrisvin/EasyReveal#1]
After adding ` 'com.github.Chrisvin:EasyReveal:1.0'` dependency i'm getting this error.

`ERROR: Unable to resolve dependency for ':app@debug/compileClasspath': Could not resolve org.jetbrains.kotlin:kotlin-stdlib:{strictly 1.3.31}.`

`ERROR: Unable to resolve dependency for ':app@debugAndroidTest/compileClasspath': Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.50.
`

**Comment by Chrisvin** [comment]
You can temporarily resolve this error by adding the following in your project build.gradle
```
ext.kotlin_version = '1.3.60-eap-25'

repositories {
        ...
        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap' }
    }
```
Take a look at the [demo project build.gradle](https://github.com/Chrisvin/EasyReveal/blob/master/build.gradle) for reference.

I didn't notice that I was using a early-access-preview version of kotlin, I'll mark this as a bug since a stable library shouldn't be using EAP dependencies.
Kindly refrain from using this in production code until this issue is resolved (probably within 24 hours), thanks for the patience. :)

**Comment by Chrisvin** [comment]
Also, if you're unable to temporarily resolve the error using the EAP kotlin version (as mentioned above), then kindly share your build.gradle files.

**Comment by sivabe35** [comment]
Thanks... After adding EAP dependencies Now its working fine.

**Comment by Chrisvin** [comment]
Turns out that the actual issue was the dokka plugin. It used to resolve it's dependency from the project build.gradle (which explains why it was temporarily fixed by adding the EAP maven repo).

This has now been fixed with [release 1.2](https://github.com/Chrisvin/EasyReveal/releases/tag/1.2)

**Comment by Chrisvin** [comment]
Feel free to reopen if the problem persists or reappears.
